agent/agent.go

```diff
@@ -546,9 +552,9 @@ func customSeekers(cfg *ProductConfig, tmpDir string) ([]*seeker.Seeker, error)
 	var err error
 	switch cfg.Name {
 	case product.Consul:
-		c = client.NewConsulAPI()
+		c, err = client.NewConsulAPI()
 	case product.Nomad:
-		c = client.NewNomadAPI()
+		c, err = client.NewNomadAPI()
```

Comment:
Ensure err is handled

Reply:
I think it should be handled on [line 563](https://github.com/hashicorp/hcdiag/pull/103/files/896abcf9658fda690a0fd9dfeb6e0887f4aaca29#diff-62b6ad581fe3a3059ae8c85ef0f31dde4092bfdecfa7d6857c470bcacaa8cc8bR563).